sema/commons/j2/j2_functions.py

```
@@ -24,20 +25,22 @@ def all():
         }
 
 
-def xsd_value(content, quote, type_name, suffix=None):
+def xsd_value(content: any, quote: str, type_name: str, suffix: str | None = None) -> str:
     if suffix is None:
         suffix = "^^" + type_name
     return quote + str(content) + quote + suffix
 
 
-def xsd_format_boolean(content, quote, suffix):
+def xsd_format_boolean(content: any, quote: str, suffix: str) -> str:
```

Ruff (0.8.2)

Unused function argument: `suffix`

(ARG001)